lib/Command/ActiveCalls.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2020 Joas Schilling <coding@schilljs.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Talk\Command;
+
+use OC\Core\Command\Base;
+use OCA\Talk\Manager;
+use OCA\Talk\Participant;
+use OCP\IDBConnection;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ActiveCalls extends Base {
+
+	/** @var IDBConnection */
+	public $connection;
+
+	/** @var Manager */
+	public $manager;
+
+	public function __construct(IDBConnection $connection) {
+		parent::__construct();
+
+		$this->connection = $connection;
+	}
+
+	protected function configure(): void {
+		$this
+			->setName('talk:active-calls')
+			->setDescription('Allows you to check if calls are currently in process');
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output): ?int {
+		$query = $this->connection->getQueryBuilder();
+
+		$query->select($query->func()->count('*', 'num_calls'))
+			->from('talk_rooms')
+			->where($query->expr()->isNotNull('active_since'));
+
+		$result = $query->execute();
+		$numCalls = (int) $result->fetchColumn();
+		$result->closeCursor();
+
+		if ($numCalls === 0) {
+			$output->writeln('<info>No calls in progress</info>');
+			return 0;
+		}
+
+		$query->select($query->func()->count('*', 'num_participants'))
+			->from('talk_participants')
+			->where($query->expr()->gt('in_call', $query->createNamedParameter(Participant::FLAG_DISCONNECTED)))
+			->andWhere($query->expr()->neq('session_id', $query->createNamedParameter('0')))
+			->andWhere($query->expr()->gt('last_ping', $query->createNamedParameter(time() - 60)));
```

Comment:
The count of participants in the call does not seem to be right; I started a call with just one participant and the count returned three, and when I started a second call the participants count went up to six. Note that is not always a multiple of three, though; at another point during the tests it returned eight participants.

Reply:
The count was correct, the user placeholder was not